### PR TITLE
update README to mention that the content has been reintegrated into …

### PR DESCRIPTION
**Pull request by dirk-thomas**
Related to #492.

~~Before merging the number of the ros/rosdistro PR needs to be inserted...~~ Done.

### CODE REVIEW

**Comment by nuclearsandwich** [comment]
Removing or renaming the index.yaml and index-v4.yaml files would prevent people from unintentionally targeting this repository with bloom PRs. Is that something we want to incorporate into this PR?

**Comment by dirk-thomas** [comment]
Absolutely, similar to other archived repos (e.g. https://github.com/ros2/console_bridge) I removed all the content in 120015f6d3a64643a1a43536ba4a505e13d1b7bd.

File: README.md [hunk 1]
```diff
@@ -1,26 +1,2 @@
...
-To create jobs on our build farm to build Debian sourcedeb and binarydeb packages add your bloom (or git-buildpackage compatible) repository to the `ROSDISTRO/distribution.yaml` file.
-Please keep the alphabetic order of the list.
-
-Files and directories:
-
- - index.yaml: list of ROS distributions (Groovy and newer) with references to their release, source and doc files
- - ROSDISTRO: the distribution files of each ROS distribution and their corresponding build files
- - scripts: support scripts
-
+The content of this repository is no more being used and has been reintegrated into the upstream repository [ros/rosdistro#19908](https://github.com/ros/rosdistro/pull/19908).
```
Comment (nuclearsandwich) [inline README.md]: I suggest rewording this. In addition to being simpler English. Two separate sentences are easier for automatic translation to handle than a compound sentence.

```suggestion
This repository is no longer in use.
The content of this repository has been reintegrated into the upstream repository [ros/rosdistro#19908](https://github.com/ros/rosdistro/pull/19908).
```

**Review by nuclearsandwich — APPROVED** [review]
LGTM with one suggestion.

**Review by jacobperron — APPROVED** [review]
